packages/graphql-language-service-config/src/GraphQLConfig.js

```diff
@@ -16,7 +16,7 @@ import type {
 
 import path from 'path';
 
-const APP_EXTENSIONS_NAME = 'extensions';
+const APP_EXTENSIONS_NAME = 'projects';
```

Comment:
Should probably change the variable name too (`APP_EXTENSIONS_NAME` → `PROJECTS_NAME` or something like that?)